tests/codebuild/run_integtest.sh

```diff
@@ -78,40 +103,101 @@ else
     readonly cluster_region="$(echo "${cluster_info}" | awk '{print $2}')"
 fi
 
-
+echo "SETUP FOR INTEGRATION TESTS"
 # Download the CRD from the tarball artifact bucket
 aws s3 cp s3://$ALPHA_TARBALL_BUCKET/${CODEBUILD_RESOLVED_SOURCE_VERSION}/sagemaker-k8s-operator-us-west-2-alpha.tar.gz sagemaker-k8s-operator.tar.gz 
 tar -xf sagemaker-k8s-operator.tar.gz
-
 # Jump to the root dir of the operator
 pushd sagemaker-k8s-operator
-
-    # Setup the PATH for smlogs
+# Setup the PATH for smlogs
     mv smlogs-plugin/linux.amd64/kubectl-smlogs /usr/bin/kubectl-smlogs
-
-    # Allow for overriding the installation of the CRDs/controller image from the
-    # build scripts if we want to use our own installation
-    if [ "${SKIP_INSTALLATION}" == "true" ]; then
-        echo "Skipping installation of CRDs and operator"
-    else
-        # Goto directory that holds the CRD  
-        pushd sagemaker-k8s-operator-install-scripts
-            # Since OPERATOR_AWS_SECRET_ACCESS_KEY and OPERATOR_AWS_ACCESS_KEY_ID defined in build spec, we will not create new user
-            ./setup_awscreds
-
-            echo "Deploying the operator"
-            kustomize build config/default | kubectl apply -f -
-        popd
-
-        echo "Waiting for controller pod to be Ready"
-        # Wait to increase chance that pod is ready
-        # TODO: Should upgrade kubectl to version that supports `kubectl wait pods --all`
-        sleep 60
-    fi 
+    pushd sagemaker-k8s-operator-install-scripts
+        # Since OPERATOR_AWS_SECRET_ACCESS_KEY and OPERATOR_AWS_ACCESS_KEY_ID defined in build spec, we will not create new user
+        ./setup_awscreds
```

Comment:
I am under impression that pod can assume role, if so why do we still need this?

Reply:
because i have not modified the cluster scope operator to use the IAM role, i think that can be done as a separate task and increases the scope of this one even further.

Reply:
I am confuse, if operator has both `IAM-Role` and `Credentials` which one it will use ? So either comment this or we need to make necessary change. 

I am saying because there could be bug in `IAM Role` implementation which we might miss.

Reply:
The cluster scope operators are using credentials and the namespaced scope are using IAM role. 
migrating the clusterscoped operator to use the IAM role is a separate task and at that point we can get rid of credentials completely. 
Right now I can find a way to clean up the credentials before we start the namespace deployment tests. However I assume this would require some more refactoring to clean up properly.

Reply:
but in namespace scope deployment do we use this code ? I think we do.

Reply:
No. I commented out the cluster scoped section including the ` ./setup_awscreds` and deleted the existing user. The namespaced test ran just fine.

Reply:
Assuming that after this change the installation will be namespace scoped which will use IAM role? 

Also do create a task in background to add support for cluster scope installation with IAM role.

Reply:
Not sure what you mean. The local test i ran without setup_awscreds was for namespaced installation and tests. 
The integ tests do the folllowing - 
1. setup awscreds
2. Install clusterscoped operator
3. run cluster scoped tests
4. delete cluster scoped operator
5. create namespaced OIDC based IAM role
6. use #5 to deploy namespaced operator
7. Run namespaced tests. 

What i meant above was that i can run steps 5, 6, 7 without needing the aws credentials.